Source/JavaScriptCore/runtime/ArrayBuffer.cpp

```diff
@@ -268,6 +276,12 @@ RefPtr<ArrayBuffer> ArrayBuffer::tryCreate(const void* source, size_t byteLength
     return createInternal(WTFMove(contents), source, byteLength);
 }
 
+RefPtr<ArrayBuffer> ArrayBuffer::tryCreate(std::span<uint8_t>&& span)
+{
+    auto s = std::exchange(span, { });
```

Comment:
This whole line is not needed.

Reply:
> The thing I like about using a reference is that it can be explicitly cleared out so that if someone tries to use it afterwards, it will produce an error, rather than working "by accident".